### ISSUE TITLE
Container stopping with `/usr/local/bin/exabgp: No such file or directory`

### ISSUE THREAD

**Issue by patrickhanft** [exacheck/exacheck#13]
Hi,

I recently stumbled over this project and became very curious. Thank you for all your nice work!

I wanted to try it out and I have to admit my docker knowledge could really improve (that's why I am trying to do more with it), but when trying to deploy the docker container I got this error, both when trying to use docker compose:

```sudo docker compose up
[+] Running 0/1
 ⠙ Container exacheck  Recreated                                                                 0.1s 
Attaching to exacheck
exacheck  | [dumb-init] /usr/local/bin/exabgp: No such file or directory
exacheck exited with code 2
```
but also when trying to build the container myself:

```
vagrant@ubuntu:~/exacheck/docker$ sudo docker run 6ec13e6ab94d8b86199b95e4a3a9ad0c889333d294f051f001c3a62b517dd96c
[dumb-init] /usr/local/bin/exabgp: No such file or directory
```

Did I misunderstand something, because I thought that the exacheck container should bring its own exabgp, shouldn't it? I'd be very happy about any hints, how I can fix that myself. Thank you!

**Comment by gbe0** [comment]
Hi,

Thanks!

I see the problem, I will apply a fix shortly and release v0.1.1. I made a mistake after I refactored a bunch of things and it left ExaBGP itself out of the Docker build.

**Comment by gbe0** [comment]
Hi,

This should now be resolved in [v0.1.1](https://github.com/exacheck/exacheck/releases/tag/v0.1.1). The images have been pushed to Docker Hub; please give it a try and let me know if you have further issues.

Also if you have any feature requests or need help feel free to create a new issue or contact me directly: exacheck@exacheck.net.

Thanks!

**Comment by patrickhanft** [comment]
OMG, I should have found more time earlier to revisit this topic. This fix was REALLY fast! It works now for me. Thank you for this very, very rapid fix!
